src/commit.rs

```diff
@@ -87,3 +87,10 @@ pub fn pre_commit_check(pre_commit_command: Option<String>, message: &str) -> Re
     }
     Ok(())
 }
+
+pub fn git_add_all_modified() -> Result<()> {
```

Comment:
Something easier would be putting the `-a` flag directly on commit command execution here:

https://github.com/alt-art/commit/blob/main/src/commit.rs#L31

Reply:
hmm... wrap the check_staged_files in a if

Reply:
If all, there is no need to run at all

Reply:
Not sure about the behaviour then if I run it with -a but no file was modified. Maybe I would need to do another check in commit then?

Reply:
I think it is okay to leave in that away for now.